### ISSUE TITLE
zIndex bug for absolute view

### ISSUE THREAD

**Issue by melkayam92** [grahammendick/navigation#315]
Before I used the libary I could do in my scene:

```
<View>
   <View style={{position: 'absolute', zIndex: 20}}>
       <Autocomplete />
  </View>

  <Flatlist/>

</View>
```

The autocomplete can open above the list. now what happens is that the Flatlist is always above the Autocomplete view. if I swap the order of the components its works fine. (but in RN view it should be fine to use the zIndex as above)

Any idea why it happens?  both iOS and Android so I guess it's not related to something native.

**Comment by melkayam92** [comment]
I created this simple example to demonstrate: 

import React from 'react';
import { View } from 'react-native';

```
export default function Test() {
  return (
    <View style={{ flex: 1 }}>
      <View
        style={{ backgroundColor: 'green', width: 50, height: 50, position: 'absolute', zIndex: 20 }}
      ></View>

      <View style={{ backgroundColor: 'red', width: 200, height: 200 }}></View>
    </View>
  );
}
```

<img width="392" alt="Screen Shot 2019-10-20 at 21 59 34" src="https://user-images.githubusercontent.com/8774744/67164606-00bb8500-f385-11e9-9138-1a5a4b4499c8.png">

The top half of the screen is View that not under the Navigator, and the bottom half is the one that does inside.

I was thinking maybe the reason is the the Scene component is absolute?

**Comment by melkayam92** [comment]
After testing again, in iOS, it seems like its working fine. the problem is only on Android.

I guess its something to do with the FragmentContainer.

**Comment by grahammendick** [comment]
Could you retry your code with the `collapsable` prop set to false, please? It works for me. 
```jsx
<View style={{ flex: 1 }} collapsable={false}>
  <View style={{ backgroundColor: 'green', width: 50, height: 50, position: 'absolute', zIndex: 20 }}></View>
  <View style={{ backgroundColor: 'red', width: 200, height: 200 }}></View>
</View>
```

**Comment by melkayam92** [comment]
I will check it..

But it means that we lose here significant performance of how RN layout views in Android.

Maybe it's worth check if we can fix it's in the Android native side.

**Comment by grahammendick** [comment]
You only need it on the very first child of the Scene. You don't need it on any other views.
The chances are the first child is a `SafeAreaView` or a `ScrollView` , in which case you don't need it at all.

**Comment by melkayam92** [comment]
I see.

I will check it in my real use case and let you know.

**Comment by melkayam92** [comment]
It looks like its working! thanks!!

**Comment by grahammendick** [comment]
Great. Happy to help

**Comment by grahammendick** [comment]
Hey @melkayam92 while working on [something else](https://github.com/grahammendick/navigation/pull/533) I worked out what was wrong. The `SceneView` java class has to inherit from `ReactViewGroup` (instead of `ViewGroup`) to get the zIndex working. I've just released the fix in navigation-react-native v8.1.1
